### ISSUE TITLE
Validation loss is lower than the training loss for training on custom dataset

### ISSUE THREAD

**Issue by khaghanijavad** [MIC-DKFZ/nnUNet#931]
Hello, 

Thank you very much for your interesting work. 
I am training nnU-Net on a custom dataset, but the validation loss is lower than the training loss. I am using the 5 fold cross validation default settings. 
Do you think that this happens due to effect of regularization terms/ layers (or even sampling a random set of patches for rapid validation) or I am doing something wrong?
Do you have similar experience during training on other datasets? 

The progress.png file is attached here. Thank you very much 

![progress](https://user-images.githubusercontent.com/44342805/151527085-1c639480-7002-4d2f-9c9b-78441a483d90.png)

**Comment by FabianIsensee** [comment]
Hi,
this is normal. Some datasets, especially small ones do this sometimes. Do you see this behavior on all 5 folds? My guess is that it's like that just for some of the folds.
Let the training finish, the blue line will catch up ;-) 
Best,
Fabian

**Comment by khaghanijavad** [comment]
Thank you very much for your quick reply, Fabian. 
I am running the training over the first fold of data.
Thank you for your comments.

Sincerely,
Javad
